### PR TITLE
Add info on Grafana dashboards

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/giantswarm/issues/17157

This adds documentation for Grafana dashboards being accessible for customers.

### CODE REVIEW

File: src/content/general/overview/index.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ menu:
     identifier: general-overview
     parent: general
 last_review_date: 2020-06-17
+user_questions:
+  - What's included with Kubernetes run by Giant Swarm?
+  - What's not included with Kubernetes run by Giant Swarm?
```
Comment (marians) [inline src/content/general/overview/index.md:13]: Just some boyscouting

File: src/content/general/overview/index.md [hunk 2]
```diff
@@ -38,7 +41,7 @@ Your clusters comes out-of-the-box as follows:
 
 There are some things not included in the cluster as managed by us:
 
-- Additional user-space services like Dashboard, Monitoring, Logging, Registry, etc. are not installed (getting them running is [really easy]({{< relref "/app-platform/overview" >}}), though).
+- Additional user-space services like logging, container registry, etc. are not installed (getting them running is [really easy]({{< relref "/app-platform/overview" >}}), though).
```
Comment (marians) [inline src/content/general/overview/index.md:45]: Removing the info that we don't offer dashboards

**Comment by marians** [comment]
@JosephSalisbury I'd be happy if you could review this both for language and for content.

File: src/content/general/overview/index.md (hunk 2 above)
Comment (JosephSalisbury) [reply src/content/general/overview/index.md:45]: I'm not certain I agree here - I wouldn't say our new dashboard work is user-space. i.e: you would still need to install dashboards and monitoring for any of your applications (to start with, at least)

File: src/content/ui-api/monitoring/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+For accessing these metrics, both for you as customers and for our staff we provide a [Grafana](https://grafana.com/) instance per installation.
+
+![Home dashboard](home-dashboard.png)
+
+We aim to provide a useful set of dashboards for you to quickly explore all metrics relevant to you, giving you access to the same information that we use for operations, support, and billing.
+
+## Accessing dashboards
+
+If you have access to the Giant Swarm [web interface]({{< relref "/ui-api/web" >}}), the easiest way to get to Grafana is to click the link **Monitoring** in the top menu when logged in.
```
Comment (JosephSalisbury) [inline src/content/ui-api/monitoring/index.md:28]: random brain wave: weren't you working on something that would show people their base domain URLs or something? it'd be really cool to directly link from docs to their installation's grafana

File: src/content/ui-api/monitoring/index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+![Selecting an OIDC provider](access.png)
+
+There is one identity provider configured for Giant Swarm staff and one for you as a customer admin. Depending on the type of identity provider used for you as a customer, the label for the button to click can look slightly different.
+
+After selecting the right identity provider, you may run through an authentication workflow. However, if you are already authenticated within the current browser, this will be skipped and you should see the Home dashboard as a result.
+
+## Repositories
+
+Grafana is deployed to Giant Swarm management clusters via our own App, to be found in the [giantswarm/g8s-grafana](https://github.com/giantswarm/g8s-grafana) Github repository.
```
Comment (JosephSalisbury) [inline src/content/ui-api/monitoring/index.md:60]: Is it useful to link to a private repo?

File: src/content/ui-api/monitoring/index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+Dashboard are defined in the [giantswarm/dashboards](https://github.com/giantswarm/dashboards) repository. You can easily track changes in that repository directly from the Home dashboard in Grafana or via the [changes and releases](/changes/dashboards/) section.
+
+## Limitations
+
+Grafana access requires an **exclusive installation** and is currently not available in shared installations, where several customer's metrics are collected using the same Prometheus instance.
```
Comment (JosephSalisbury) [inline src/content/ui-api/monitoring/index.md:66]: Is 'exclusive installation' new jargon? do we need to explain it further?

**Comment by JosephSalisbury** [comment]
also requested from atlas to have more eyes :D

File: src/content/ui-api/monitoring/index.md (hunk 3 above)
Comment (marians) [reply src/content/ui-api/monitoring/index.md:28]: Yes, but it's far from done.

File: src/content/ui-api/monitoring/index.md (hunk 4 above)
Comment (marians) [reply src/content/ui-api/monitoring/index.md:60]: No. I missed that part. Will remove that bit.

File: src/content/ui-api/monitoring/index.md (hunk 5 above)
Comment (marians) [reply src/content/ui-api/monitoring/index.md:66]: I'll simplify that

**Comment by QuentinBisson** [comment]
I really like what you did Marian :)

File: src/content/general/overview/index.md (hunk 2 above)
Comment (marians) [reply src/content/general/overview/index.md:45]: I see

**Comment by marians** [comment]
Is this worth an approval?

**Review by QuentinBisson — APPROVED** [review]
LGTM

File: src/content/ui-api/monitoring/index.md (hunk 3 above)
Comment (mig4) [inline src/content/ui-api/monitoring/index.md:28]:
```suggestion
If you have access to the Giant Swarm [web interface]({{< relref "/ui-api/web" >}}), the easiest way to get to Grafana is to click the **Monitoring** link in the top menu when logged in.
```

File: src/content/ui-api/monitoring/index.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+**Note**: If you don't have SSO configured yet or have any questions regarding the admin group, please contact your Account Engineer.
+
+Once you open the Grafana URL for your installation, you will be greeted by a page titled "Log in to dex", where you are prompted to select the identity provider to authenticate with. The screenshot below shows an example.
+
+![Selecting an OIDC provider](access.png)
+
+There is one identity provider configured for Giant Swarm staff and one for you as a customer admin. Depending on the type of identity provider used for you as a customer, the label for the button to click can look slightly different.
+
+After selecting the right identity provider, you may run through an authentication workflow. However, if you are already authenticated within the current browser, this will be skipped and you should see the Home dashboard as a result.
```
Comment (mig4) [inline src/content/ui-api/monitoring/index.md:56]:
```suggestion
After selecting the right identity provider, you may run through an authentication workflow, unless you are already authenticated within the current browser, then you should see the Home dashboard.
```

File: src/content/ui-api/monitoring/index.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+In order to access Grafana as a customer, you must be a member of the admin group. Specifically, that's a group in your identity provider selected to specify which users have admin permissions in the Giant Swarm installation.
+
+**Note**: If you don't have SSO configured yet or have any questions regarding the admin group, please contact your Account Engineer.
+
+Once you open the Grafana URL for your installation, you will be greeted by a page titled "Log in to dex", where you are prompted to select the identity provider to authenticate with. The screenshot below shows an example.
+
+![Selecting an OIDC provider](access.png)
+
+There is one identity provider configured for Giant Swarm staff and one for you as a customer admin. Depending on the type of identity provider used for you as a customer, the label for the button to click can look slightly different.
```
Comment (mig4) [inline src/content/ui-api/monitoring/index.md:54]: Maybe?

```suggestion
There is one identity provider configured for Giant Swarm staff and one for you as a customer admin. Depending on the type of identity provider used by your organization, the label for the button to click can look slightly different.
```

**Review by mig4 — APPROVED** [review]
Just a couple of suggestions, but LGTM.

**Pull request merged**